piwik-sdk/src/test/java/org/piwik/sdk/TrackerTest.java

```diff
@@ -808,6 +809,19 @@ public void testPreviousVisit() throws Exception {
         assertEquals(1000l, Long.parseLong(queryParams.get(QueryParams.PREVIOUS_VISIT_TIMESTAMP)));
     }
 
+    @Test
+    public void testSetCustomDimensions() throws Exception {
+        CustomDimensions customDimensions = new CustomDimensions();
+        customDimensions.set(1, "foo");
+        customDimensions.set(2, "bar");
+        customDimensions.set(3, "empty").set(3, null);
```

Comment:
Good check!
What is the expected behavior client/server side for an empty string ```""``` ?